python-backend/app/scheduled_jobs/NRIS_jobs.py

```diff
@@ -0,0 +1,46 @@
+from app.extensions import cache, sched
+from app.api.nris_services import NRIS_service
+from app.api.mines.mine.models.mine import Mine
+from app.api.constants import NRIS_JOB_PREFIX, NRIS_MMLIST_JOB, NRIS_MAJOR_MINE_LIST, TIMEOUT_24_HOURS, TIMEOUT_60_MINUTES
+
+
+#the schedule of these jobs is set using server time (UTC)
+def _schedule_NRIS_jobs(app):
+    app.apscheduler.add_job(
+        func=_cache_major_mines_list, trigger='cron', id='get_major_mine_list', hour=9, minute=0)
+    app.apscheduler.add_job(
+        func=_cache_all_NRIS_major_mines_data,
+        trigger='cron',
+        id='get_major_mine_NRIS_data',
+        hour=9,
+        minute=10)
+
+
+# caches a list of mine numbers for all major mines and each major mine indavidually
+# to indicate whether of not it has been processed.
+def _cache_major_mines_list():
+    if sched.app.config.get('ENVIRONMENT_NAME') == 'prod':
```

Comment:
Do you need this check here in addition to the one in init.py?

Reply:
Not any more. I added the check in the init because I was getting a strange issue in the test phase where they were all failing because the scheduler was being started for every test.